### ISSUE TITLE
Severe performance issues in WinUI 3

### ISSUE THREAD

**Issue by BitCrusader** [microsoft/microsoft-ui-xaml#6031]
### Describe the bug

Getting some brutal performance issues in all versions of WinUI 3, including the latest 1.0 Preview 1. Running the following on CompositionTarget.Rendering with a completely empty window consistently yields results between 29 and 31ms:
```cs
PerformanceCounter.Stop(); //PerformanceCounter is an instance of class Stopwatch
Console.WriteLine(PerformanceCounter.Elapsed.TotalMilliseconds + "ms");
PerformanceCounter.Restart();
```

For comparison, if I do exactly the same in UWP XAML I get results in the ballpark of 0.5ms - that's 60 times faster than WinUI 3, which's current framerate (equivalent to ~33fps) is not acceptable even for a desktop UI.

### Steps to reproduce the bug
```cs
public App()
{
    InitializeComponent();
    CompositionTarget.Rendering += PerformanceTest;
}

private Stopwatch PerformanceCounter = new();
private void PerformanceTest(object Sender, object Event)
{
    PerformanceCounter.Stop();
    Console.WriteLine(PerformanceCounter.Elapsed.TotalMilliseconds + "ms");
    PerformanceCounter.Restart();
}
```

### Expected behavior

At minimum, I expect the average user's PC to produce at least 60fps (16.6ms) in an empty window, ideally being able to match any common refresh rate. As shown by the tests above, a PC with considerably above-average specifications produces framerates that are well below what the average user that would consider acceptable.

### Windows app type

- [ ] UWP
- [X] Win32

### Device form factor

Desktop

### Windows version

Insider Build (22000.194)

### Additional context

Tested on a machine with the following specs:
OS: Windows 11 Pro (22000.194)
GPU: NVIDIA GeForce GTX 1080ti
CPU: AMD Ryzen 5 2600
RAM: 2x8GB DDR4, 3000MHz

**Comment by StephenLPeters** [comment]
@MikeHillberg and @codendone

**Comment by jeremy-visionaid** [comment]
Well over a year now, and WinUI 3 rendering perf is still too slow to consider it being usable for certain classes of application :(
Really wish it were open source so the community could fix it themselves...

**Comment by bpulliam** [comment]
A fix for the core issue is included in the 1.3 release, released April 12th. If you are still seeing low framerates on high refresh monitors using 1.3, please log a new issue, as it is likely a different scenario and we should track it separately.

**Comment by jeremy-visionaid** [comment]
@bpulliam While I'm glad it's finally fixed, this really shouldn't have taken 18 months to get to the point where WinUI 3 could be used for basically expected stuff like playing videos or camera previews (which also meant the same for MAUI apps BTW). 

I think some kind of project management retrospective may be in order to find out what went wrong here. It may have technically just been a performance issue, but one serious enough that our company couldn't commit to developing on the platform. Even more concerning than the bug itself was the lack of acknowledgement that this was even being looked at, which is particularly important given the community's inability to deal with these things themselves.
